docs/esa/jwst/jwst.rst

```diff
@@ -423,97 +406,70 @@ Query without saving results in a file:
   >>> from astroquery.esa.jwst import Jwst
   >>>
   >>> job=Jwst.launch_job("SELECT TOP 100 \
-  >>> instrument_name, observationuri, planeid, calibrationlevel, \
+  >>> instrument_name, proposal_id, calibrationlevel, \
   >>> dataproducttype \
   >>> FROM jwst.main ORDER BY instrument_name, observationuri")
```

Comment:
Instead of using backslashes PEP 8 recommends implied line continuation inside parentheses:
```python
job = Jwst.launch_job("SELECT TOP 100 "
    "instrument_name, proposal_id, calibrationlevel "
    "dataproducttype "
    "FROM jwst.main ORDER BY instrument_name, observationuri")
```
But triple quotes might also be acceptable:
```python
job = Jwst.launch_job("""SELECT TOP 100 
    instrument_name, proposal_id, calibrationlevel 
    dataproducttype 
    FROM jwst.main ORDER BY instrument_name, observationuri""")
```